Samples/src/SimpleGrover/README.md

```diff
@@ -0,0 +1,14 @@
+﻿# Grover Search Sample #
+
+This sample implements Grover's search algorithm. Oracles implementing the database are explicitly constructed together with all steps of the algorithm.
+
+## Running the Sample in Visual Studio ##
+
+Open the `QsharpSamples.sln` solution in Visual Studio and set the .csproj file in the manifest as the startup project.
+Press Start in Visual Studio to run the sample.
```

Comment:
I'd suggest adding a `host.py` along with instructions on how to run the sample from the command line using either Python or the .NET Core SDK:

````suggestion
Press Start in Visual Studio to run the sample.

## Running the Sample from the Command Line ##

This sample can also be run from the command line, including from the integrated terminal in Visual Studio Code.
To run the C# host program for this sample, use the `dotnet` command:

```bash
dotnet run
```

To run this sample from Python:

```bash
python host.py
```
````